script/excludes/it.php

```diff
@@ -11,9 +11,135 @@
     |
     */
 
+    'ID',
     'Email',
     'Oh no',
     'Editor',
     'Password',
     'Dashboard',
+    'No',
+
+    /**
+     * Countries
```

Comment:
The country names do not need to be specified in the exclusion list, which is needed to generate the todo.md file.

The list of countries is already listed in the global list for all localizations: [script/excludes/_all.php](https://github.com/Laravel-Lang/lang/blob/master/script/excludes/_all.php)

Reply:
ok